tests/test_ops/test_roi_align.py

```diff
@@ -94,7 +92,16 @@ def _test_roialign_allclose(device, dtype):
             x.grad.data.type(torch.float).cpu().numpy(), np_grad, atol=1e-3)
 
 
-@pytest.mark.parametrize('device', ['cuda', 'cpu'])
+@pytest.mark.parametrize('device', [
+    'cpu',
+    pytest.param(
+        'cuda',
+        marks=pytest.mark.skipif(
+            not is_cuda(), reason='requires CUDA support')),
+    pytest.param(
+        'mlu',
+        marks=pytest.mark.skipif(not is_mlu(), reason='requires MLU support'))
+])
 @pytest.mark.parametrize('dtype', [torch.float, torch.double, torch.half])
```

Comment:
double test failed.

Reply:
It will skip when data type is double on MLU.